tests/test_utils.py

```diff
@@ -0,0 +1,40 @@
+from odfuzz.utils import encode_string, decode_string
```

Comment:
This is a good unit test for the utils method, that's OK.

But ideally we should have tests for all EDM data types with the current encoder/decoder on and off. If I am correct, the issue #40 is not yet reproduced by any existing tests and current generator and mutator tests don't handle the encoding even that both classes has been changed in this PR.

Reply:
The issue may be reproduced by running the integration test and verifying whether a generated URL contains the string in the format `datetime''XXXX-XX-XX''` or not. Otherwise, if you want me to create a new test case specifically for this scenario, I can try to do it.